src/iceberg/sort_field.h

```diff
@@ -41,13 +43,52 @@ enum class SortDirection {
   /// Descending
   kDescending,
 };
+/// \brief Get the relative sort direction name
+ICEBERG_EXPORT constexpr std::string_view SortDirectionToString(SortDirection direction) {
```

Comment:
nit: we should probably only declare these functions here and put all implementation in the .cc file.